deploy/operator.yaml

```diff
@@ -44,23 +44,23 @@ spec:
           imagePullPolicy: Always
           env:
             - name: ICP_PLATFORM_AUTH_IMAGE
-              value: quay.io/opencloudio/icp-platform-auth:3.7.5
+              value: quay.io/opencloudio/icp-platform-auth:3.7.6
             - name: ICP_IDENTITY_PROVIDER_IMAGE
-              value: quay.io/opencloudio/icp-identity-provider:3.7.5
+              value: quay.io/opencloudio/icp-identity-provider:3.7.6
             - name: ICP_IDENTITY_MANAGER_IMAGE
-              value: quay.io/opencloudio/icp-identity-manager:3.7.5
+              value: quay.io/opencloudio/icp-identity-manager:3.7.6
             - name: IAM_POLICY_ADMINISTRATION_IMAGE
-              value: quay.io/opencloudio/iam-policy-administration:3.7.5
+              value: quay.io/opencloudio/iam-policy-administration:3.7.6
             - name: IAM_POLICY_DECISION_IMAGE
-              value: quay.io/opencloudio/iam-policy-decision:3.7.5
+              value: quay.io/opencloudio/iam-policy-decision:3.7.6
             - name: ICP_SECRET_WATCHER_IMAGE
-              value: quay.io/opencloudio/icp-secret-watcher:3.7.5
+              value: quay.io/opencloudio/icp-secret-watcher:3.7.6
             - name: ICP_OIDCCLIENT_WATCHER_IMAGE
-              value: quay.io/opencloudio/icp-oidcclient-watcher:3.7.5
+              value: quay.io/opencloudio/icp-oidcclient-watcher:3.7.6
             - name: IAM_POLICY_CONTROLLER_IMAGE
-              value: quay.io/opencloudio/iam-policy-controller:3.7.5
+              value: quay.io/opencloudio/iam-policy-controller:3.7.6
             - name: ICP_IAM_ONBOARDING_IMAGE
-              value: quay.io/opencloudio/icp-iam-onboarding:3.7.5
+              value: quay.io/opencloudio/icp-iam-onboarding:3.7.6
             - name: AUDIT_SYSLOG_SERVICE_IMAGE
               value: quay.io/opencloudio/audit-syslog-service:1.4.0
```

Comment:
Change to latest 1.8.0 (CS3.17 equivalent)

Reply:
done